src/reducers/components.js

```diff
@@ -18,6 +18,24 @@ const component = (state, { type, payload: {id, component, props} }) => {
 };
 
 /**
+ *
+ */
+const editProperties = (state, type, props ) => {
+    switch (type) {
+        case Actions.editComponent:
```

Comment:
Why do you need another switch statement here to check the action type?  It should already be checked before this is ever invoked.